Testscripts/Linux/nested_vm_utils.sh

```diff
@@ -42,7 +42,7 @@ Install_KVM_Dependencies()
         add_sles_network_utilities_repo
     fi
     if [ $DISTRO_NAME == "coreos" ]; then
-        LogMsg "Distro not supported. Skip the test."
+        LogErr "Distro not supported. Skip the test."
         SetTestStateSkipped
```

Comment:
@LiliDeng Please correct me if I am not. I think LogErr working with Test Abort and Test Failed only. We use Test Skipped with LogMsg. I will fix it in the document

Reply:
Updated. Thanks! I'll read How-to-use doc more carefully.

Reply:
Don't be sorry. I did not update the document. Not your fault.